static/js/app.js

```diff
@@ -473,7 +505,7 @@ function runPythonProgram(code, cb) {
 }
 
 $('#feedbackbox .expand-dialog').click(function(){
-   feedback_viewed = true;
+   feedback_viewed[feedback_level-2] = true;
```

Comment:
Maybe `feedback_level-2` is missing quotes? I don't see this variable defined anywhere.

Reply:
It's a global variable defined at line 1. At each response call it is set to the current feedback_level. Because the GFM only start from level 2 we set the index of the feedback_viewed array on true using -2.